app_dart/test/request_handlers/push_benchmark_to_center_test.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'package:cocoon_service/cocoon_service.dart';
+import 'package:cocoon_service/src/model/appengine/time_series.dart';
+import 'package:cocoon_service/src/model/appengine/time_series_value.dart';
+import 'package:metrics_center/flutter.dart' as mc;
+import 'package:test/test.dart';
+
+import '../src/datastore/fake_cocoon_config.dart';
+import '../src/request_handling/fake_logging.dart';
+
+void main() {
+  FakeConfig config;
+  FakeLogging logger;
+  PushBenchmarkToCenter handler;
+
+  setUp(() {
+    config = FakeConfig();
+    logger = FakeLogging();
+    handler = PushBenchmarkToCenter(config, logger: logger);
+  });
+
+  group('parseSinceMillis', () {
+    test('with missing parameter', () {
+      final int sinceMillis = handler.parseSinceMillis(<String, String>{});
```

Comment:
Would parseSinceMillis make more sense in the utilities class? it would be easier to reuse and I don't think it depends on the handler.